responses/meme.go

```diff
@@ -13,22 +13,26 @@ type cookiejar struct {
 	jar map[string][]*http.Cookie
 }
 
+// Used to set cookies into HTTP lib cookiejar
 func (p *cookiejar) SetCookies(u *url.URL, cookies []*http.Cookie) {
```

Comment:
Is there a reason for using `p` as the struct? An acronym like `cj` would be better imho